### ISSUE TITLE
Revival Item/Command

### ISSUE THREAD

**Issue by TrueGame776** [srnyx/limited-lives#16]
### What and How?

An item that can be (configurable recipe) crafted to revive a player (do a certain command). My config.yml of Limited Lives has the player getting banned if they are out of lives, so it should be able to do things like '/minecraft:pardon %player%', otherwise players cannot revive each other. Or even /lives give <lives> <player> should work.

### Why?

Reviving, if a person is banned by losing all their lives, is impossible.

### Are there alternatives?

No.

### Checks

- [X] I have used the search at least once to check if my idea has already been suggested and perhaps already implemented.

**Comment by YaBroBlackCat** [comment]
Real gng how do we get this noticed 😭🙏

**Comment by srnyx** [comment]
> Real gng how do we get this noticed 😭🙏

It's a bit complicated for me to do since how would the item know *who* to revive?

Maybe it could be a GUI that pops up when the item is used, it just asks them to type their username in chat, etc...
Both don't feel like very enjoyable ways for the player to revive someone, as when does an item ever open a GUI in Minecraft (i.e. it's not expected behavior)?

Other similar servers use something like the dead player's head in the crafting recipe, but I feel like most servers wouldn't want something so restrictive (you *have* to have that player's head in order to revive them, otherwise you're completely out of luck).

Let me know what you prefer or if you've got any better ideas. I'm also available on [Discord](https://srnyx.com/discord) for more active/real-time discussion!

**Comment by YaBroBlackCat** [comment]
The plugin [LifeStealz](https://modrinth.com/plugin/lifestealz) has a player head GUI showing all online and offline players' heads and they are all named with the player's name. If you select one, then the one you selected gets revived. There is also Discord webhook integration, so if someone is revived, there is a ping in the server.

**Comment by srnyx** [comment]
> The plugin [LifeStealz](https://modrinth.com/plugin/lifestealz) has a player head GUI showing all online and offline players' heads and they are all named with the player's name. If you select one, then the one you selected gets revived. 

Yeah that's what I would do if I make it a GUI, but I just don't like the idea of having a clunky GUI open when you right-click an item, it just doesn't make sense to me. However, I don't really see any other way to accomplish this without a GUI, so I guess I'll make a configurable recipe that gives you an item that, when clicked, opens a GUI with a list of dead players.

I really would've loved to find a better way to implement this that didn't require searching through a GUI to find the player you wanted to revive :cry:

> There is also Discord webhook integration, so if someone is revived, there is a ping in the server.

If you want me to add that, create a new issue/suggestion please. I would make it hook with DiscordSRV rather than its own webhook.
